src/containers/EditListingPage/EditListingWizard/EditListingWizard.js

```diff
@@ -489,7 +489,7 @@ class EditListingWizard extends Component {
 
     // Redirect user to first tab when encoutering outdated draft listings.
     if (invalidExistingListingType && isNewListingFlow && selectedTab !== tabs[0]) {
-      return <NamedRedirect name="EditListingPage" params={{ ...params, tab: tabs[0] }} />;
```

Comment:
We sure about this? Ain't this file used for the OLD editing flow and not the new one?